### PR TITLE
keyword: update for py39

### PR DESCRIPTION
**Pull request by hauntsaninja**

### CODE REVIEW

File: stdlib/2and3/keyword.pyi [hunk 1]
```diff
@@ -1,6 +1,9 @@
-# Stubs for keyword
-
+import sys
 from typing import Sequence, Text, Union
 
-def iskeyword(s: Union[Text, bytes]) -> bool: ...
+def iskeyword(s: Union[Text, str]) -> bool: ...
```
Comment (srittau) [inline stdlib/2and3/keyword.pyi:4]:
```suggestion
def iskeyword(s: Text) -> bool: ...
```

File: stdlib/2and3/keyword.pyi [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 kwlist: Sequence[str]
+
+if sys.version_info >= (3, 9):
+    def issoftkeyword(s: str) -> bool: ...
+    softkwlist: Sequence[str]
```
Comment (srittau) [inline stdlib/2and3/keyword.pyi:9]: Both are undocumented at the moment.

Comment (srittau) [reply stdlib/2and3/keyword.pyi:9]: Actually, both are not present in 3.9.0b3, are they more recent than that?

Comment (hauntsaninja) [reply stdlib/2and3/keyword.pyi:9]: yeah, it's very new: https://github.com/python/cpython/pull/20877#issuecomment-643874877

**Pull request merged**